### PR TITLE
Image location change

### PR DESCRIPTION
**Pull request by AdeHarrison**
# Motivation and Context
Image name is not consistent across travis, pom files etc...

# What has changed
build & push image as "eu.gcr.io/census-rm-ci/rm/census-rm-surveysvc"

# How to test?
**1.** check out branch

**2.** create a temporary commit by adding the following to .travis.yml file (before "- bash <(curl -s https://codecov.io/bash)" line) in "after_success:"  - 
```
- if [ "$TRAVIS_BRANCH" == "image-location-change" ] ; then
docker login -u "${DOCKER_GCP_USERNAME}" -p "${DOCKER_GCP_PASSWORD}" "${DOCKER_GCP_REGISTRY}";
docker build -t "$DESTINATION_IMAGE_NAME" .;
docker tag "eu.gcr.io/census-rm-ci/rm/census-rm-surveysvc" "eu.gcr.io/census-rm-ci/rm/census-rm-surveysvc:test";
docker push "eu.gcr.io/census-rm-ci/rm/census-rm-surveysvc:test";
fi
```
and replace "branches" section with -
```
branches:
  only:
  - master
  - image-location-change
```
**3.** push commit to trigger a travis build

**4.** check "census-rm-ci" registry for tagged image

**5.** delete tagged image

**6.** rollback temporary commit changes

# Links
[Trello](https://trello.com/c/BJQqeat8/643-fixed-image-location-name)

### CODE REVIEW

**Comment by AdeHarrison** [comment]
[Image location change](https://trello.com/c/BJQqeat8/643-image-location-change)

File: .travis.yml [hunk 1]
```diff
@@ -12,22 +12,22 @@ install: true
 go_import_path: github.com/ONSdigital/rm-survey-service
 
 script:
+  - docker login -u "${DOCKER_GCP_USERNAME}" -p "${DOCKER_GCP_PASSWORD}" "${DOCKER_GCP_REGISTRY}";
```
Comment (AdamHawtin) [inline .travis.yml]: docker login is called again below, I don't think it needs to be called here.

Comment (AdeHarrison) [reply .travis.yml]: needs to pull gcp image in integration tests

Comment (AdamHawtin) [reply .travis.yml]: 👍 makes sense, would be worth mentioning that change in the PR description.

**Pull request merged**